examples/pulse_control/4_Bell_pair_with_pulses_Rigetti.ipynb

```diff
@@ -0,0 +1,539 @@
+{
```

Comment:
<p>To draw this <strong>device</strong> waveform <s>as it will be played on the device</s>,</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>